packages/web-client/pages/register/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import { useRouter } from 'next/router';
+import { Input, Button } from '@skillfuze/ui-components';
+import { useAlert } from 'react-alert';
+
+import RegisterSVG from '../../assets/illustrations/register.svg';
+import Layout from '../../components/Layout';
+import AuthService from '../../services/auth.service';
+
+const RegisterPage = () => {
+  const alert = useAlert();
+  const router = useRouter();
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [error, setError] = useState<any>({});
+
+  const onSubmit = async (event: React.FormEvent<HTMLFormElement>): Promise<void> => {
+    event.preventDefault();
+
+    try {
+      await AuthService.getInstance().register({
```

Comment:
Changed to AuthService.instance in login branch

Reply:
I can't pull pull right now or we'll have the same problem regarding merging a branch = merging the other.
Will fix it later.